src/main/java/mat/client/measure/ManageMeasurePresenter.java

```diff
@@ -1458,6 +1458,11 @@ public void onHistoryClicked(ManageMeasureSearchModel.Result result) {
                 displayHistory(result.getId(), result.getName());
             }
 
+            @Override
+            public void onFhirValidationClicked(ManageMeasureSearchModel.Result result) {
+                //TODO Calling MAT-354 service
```

Comment:
MAT-354 is in review already

Reply:
There is an another small story which implements this part.